### PR TITLE
Start a web-features package consisting of 1 feature

### PR DESCRIPTION
**Pull request by foolip**
That feature is Subgrid, with a spec link.

The package is in the root directory even though feature-group-definitions/ would make sense, as this simplifies Dependabot and GitHub Actions setup.

### CODE REVIEW

**Review by ddbeck — APPROVED** [review]
I've got one optional suggestion, but otherwise this looks good. Thank you!

File: build.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+import { fdir } from 'fdir';
+import stringify from 'fast-json-stable-stringify';
```
[the remark this answers is not in the document]
Comment (foolip) [reply build.ts:5]: I did this to not have to worry about which order we open the files in :)

File: package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "web-features",
+  "description": "Exploring the set of interoperable features in the Web Platform",
+  "version": "0.0.1",
```
[the remark this answers is not in the document]
Comment (foolip) [reply package.json:4]: OK, thanks!

File: build.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (ddbeck) [reply build.ts:5]: 💡

[the remark this answers is not in the document]
Comment (foolip) [reply build.ts:5]: Oh, and also we don't have to care about the order of things in the YAML files, it doesn't need to be consistent, the output will be anyway.

**Pull request merged**